app/components/navbar.js

```diff
@@ -0,0 +1,160 @@
+import image from "next/image";
+
+export default function Navbar() {
+  return (
+    <nav className="bg-orange-400">
+      <div className="mx-auto max-w-7xl px-2 sm:px-6 lg:px-8">
+        <div className="relative flex h-16 items-center justify-between">
+          <div className="absolute inset-y-0 left-0 flex items-center sm:hidden">
+            <button
+              type="button"
+              className="inline-flex items-center justify-center rounded-md p-2 text-gray-400 hover:bg-gray-700 hover:text-white focus:outline-none focus:ring-2 focus:ring-inset focus:ring-white"
+              aria-controls="mobile-menu"
+              aria-expanded="false"
+            >
+              <span className="sr-only">Open main menu</span>
+
+              <svg
+                className="block h-6 w-6"
+                fill="none"
+                viewBox="0 0 24 24"
+                stroke-width="1.5"
+                stroke="currentColor"
+                aria-hidden="true"
+              >
+                <path
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                  d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
+                />
+              </svg>
+
+              <svg
+                className="hidden h-6 w-6"
+                fill="none"
+                viewBox="0 0 24 24"
+                stroke-width="1.5"
+                stroke="currentColor"
+                aria-hidden="true"
+              >
+                <path
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                  d="M6 18L18 6M6 6l12 12"
+                />
+              </svg>
+            </button>
+          </div>
+          <div className="flex flex-1 items-center justify-center sm:items-stretch sm:justify-start">
+            <div className="flex flex-shrink-0 items-center">
+              <img
+                className="block h-8 w-auto lg:hidden"
+                src="/tickets.png"
+                alt="Your Company"
+              />
+              <img
+                className="hidden h-8 w-auto lg:block"
+                src="/tickets.png"
+                alt="Your Company"
+              />
+              <h2 className="text-2xl">Ticket-App</h2>
+            </div>
+            <div className="hidden sm:ml-6 sm:block">
+              <div className="flex space-x-4">
+                <a
+                  href="#"
+                  className="bg-black text-white rounded-md px-2 py-1.5 mt-0.5 text-sm font-medium"
```

Comment:
El boton full back opaca mucho. Habria que buscar otro color. Lo podemos ver en la call.
```suggestion
                  className="bg-black text-white rounded-md px-2 py-1.5 mt-0.5 text-sm font-medium"
```